extensions/amp-consent/0.1/consent-ui.js

```diff
@@ -232,23 +274,50 @@ export class ConsentUI {
    */
   showIframe_() {
     const {classList} = this.parent_;
+    classList.add(consentUiClasses.iframeActive);
     toggle(dev().assertElement(this.placeholder_), false);
     toggle(dev().assertElement(this.ui_), true);
-    classList.remove('loading');
-    classList.add('i-amphtml-consent-ui-in');
-    classList.add('consent-iframe-active');
+
+    // Remove transition/transform styles added by the fixed layer
```

Comment:
I'm not sure I understand this, Do we need to reapply the styles afterwards?

Reply:
In [`src/fixed-layer.js`](https://github.com/ampproject/amphtml/blob/master/src/service/fixed-layer.js#L649), styles are applied to top and transition in order to stop jank and other random bugs with static fixed elements.

However, since we want to animate our element on the fixed layer, we need to undo the styles that are initially applied to stop any weird jank or unexpected behaviors, that was we can add our animations.

Thus, the original styles do not need to be re-applied as the fixed layer service will handle that for us once we remove/add this element in the future.